akka-actor-tests/src/test/scala/akka/event/EventStreamSpec.scala

```diff
@@ -337,13 +342,26 @@ class EventStreamSpec extends AkkaSpec(EventStreamSpec.config) {
         es.subscribe(target, classOf[A]) should ===(true)
         fishForDebugMessage(a2, s"unsubscribing $target from all channels")
 
-        es.subscribe(target, classOf[A]) should ===(true)
-        fishForDebugMessage(a2, s"unsubscribing $target from all channels")
+        after(30.millis) {
```

Comment:
don't we have a more deterministic event to wait for? Or can we wrap the block into an `awaitAssert`?

Reply:
> don't we have a more deterministic event to wait for?

I don't think so

> Or can we wrap the block into an `awaitAssert`?

I'm not sure that's better?

Reply:
> I'm not sure that's better?

well, a random timeout might always fail depending on the machine running the test, `awaitAssert` should be a little more robust ...

Reply:
Ah, I guess we could put *just* the `subscribe` (but not the `fishForDebugMessage`) in the `awaitAssert`, I'm OK with that. Will update.